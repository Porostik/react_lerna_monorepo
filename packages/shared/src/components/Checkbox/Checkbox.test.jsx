import React from 'react';
import { shallow } from 'enzyme';
import { Checkbox } from './index';

describe('<Checkbox />', () => {
  //
  it('snapshot <Checkbox /> with {label}', () => {
    //
    const wrapper = shallow(<Checkbox label="test" />);

    expect(wrapper).toMatchSnapshot();
  });
});
