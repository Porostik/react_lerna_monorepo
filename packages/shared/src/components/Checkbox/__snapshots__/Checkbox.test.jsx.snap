// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> snapshot <Checkbox /> with {label} 1`] = `
<label>
  <input
    type="checkbox"
  />
  test
</label>
`;
